toml.md

```diff
@@ -888,6 +892,7 @@ name = "red delicious"
 [[fruits.varieties]]
 name = "granny smith"
 
+
```

Comment:
The extra blank line is intended to delineate the separate table elements of `fruits`.